shared/modules/types.go

```diff
@@ -48,6 +48,11 @@ type PersistenceConfig interface {
 	GetBlockStorePath() string
 	GetTxIndexerPath() string
 	GetTreesStoreDir() string
+	GetMaxConns() int32
+	GetMinConns() int32
```

Comment:
Why do we need a min?

Reply:
Min is by default 0 but added it in case we may need it in the future.